### ISSUE TITLE
ALE ghostly reporting linting issue from cocvetur

### ISSUE THREAD

**Issue by pwang2** [dense-analysis/ale#4044]
## Information

**VIM version**

:version
NVIM v0.6.0-dev+624-ga2e5c2f7c
Build type: Release

## What went wrong

Even setup as below, 
```vimscript
let g:ale_linters = {'vue': ['eslint']}
let g:ale_linters_explicit = 1
let g:ale_disable_lsp = 1
```
and command History shows no linting issue found and so does ALEDetail.  ALE message is still complaining from coc-vetur linters.  Run :ALEDisable will disappear the lint message.

![image](https://user-images.githubusercontent.com/1592014/151104563-cd01ada7-4d01-4869-a2ba-5e0d7c2202ec.png)

## Reproducing the bug
```
:PlugInstall coc.nvim 
:PlugInstall vim-ale
CocInstall coc-vetur
```

### :ALEInfo
 Current Filetype: vue
Available Linters: ['cspell', 'vls', 'volar', 'cspell', 'deno', 'eslint', 'fecs', 'flow', 'flow-language-server', 'jscs', 'jshint', 'standard', 'tsserver', 'xo']
  Enabled Linters: ['eslint']
  Ignored Linters: []
 Suggested Fixers: 
  'prettier' - Apply prettier to a file.
  'remove_trailing_lines' - Remove all blank lines at the end of a file.
  'trim_whitespace' - Remove all trailing whitespace characters at the end of every line.
 Linter Variables:

 Global Variables:

let g:ale_cache_executable_check_failures = v:null
let g:ale_change_sign_column_color = 0
let g:ale_command_wrapper = ''
let g:ale_completion_delay = v:null
let g:ale_completion_enabled = 0
let g:ale_completion_max_suggestions = v:null
let g:ale_disable_lsp = 1
let g:ale_echo_cursor = 1
let g:ale_echo_msg_error_str = 'Error'
let g:ale_echo_msg_format = '[%linter%]: %s'
let g:ale_echo_msg_info_str = 'Info'
let g:ale_echo_msg_warning_str = 'Warning'
let g:ale_enabled = 1
let g:ale_fix_on_save = 0
let g:ale_fixers = {}
let g:ale_history_enabled = 1
let g:ale_history_log_output = 1
let g:ale_keep_list_window_open = 0
let g:ale_lint_delay = 200
let g:ale_lint_on_enter = 1
let g:ale_lint_on_filetype_changed = 1
let g:ale_lint_on_insert_leave = 1
let g:ale_lint_on_save = 1
let g:ale_lint_on_text_changed = 'normal'
let g:ale_linter_aliases = {}
let g:ale_linters = {'vue': ['eslint'], 'json': ['prettier', 'jq'], 'javascript': ['eslint', 'prettier']}
let g:ale_linters_explicit = 1
let g:ale_linters_ignore = {}
let g:ale_list_vertical = 0
let g:ale_list_window_size = 10
let g:ale_loclist_msg_format = '[%linter%]: %s'
let g:ale_max_buffer_history_size = 20
let g:ale_max_signs = -1
let g:ale_maximum_file_size = v:null
let g:ale_open_list = 0
let g:ale_pattern_options = v:null
let g:ale_pattern_options_enabled = v:null
let g:ale_root = {}
let g:ale_set_balloons = 0
let g:ale_set_highlights = 1
let g:ale_set_loclist = 1
let g:ale_set_quickfix = 0
let g:ale_set_signs = 1
let g:ale_sign_column_always = 0
let g:ale_sign_error = '>>'
let g:ale_sign_info = '--'
let g:ale_sign_offset = 1000000
let g:ale_sign_style_error = '>>'
let g:ale_sign_style_warning = '--'
let g:ale_sign_warning = '--'
let g:ale_sign_highlight_linenrs = 0
let g:ale_statusline_format = v:null
let g:ale_type_map = {}
let g:ale_use_global_executables = v:null
let g:ale_virtualtext_cursor = 0
let g:ale_warn_about_trailing_blank_lines = 1
let g:ale_warn_about_trailing_whitespace = 1
  Command History:

(executable check - success) eslint_d
(finished - exit code 0) ['/usr/bin/zsh', '-c', 'cd ''/home/pwang2/app-client'' && ''eslint_d'' -f json --stdin --stdin-filename ''/home/pwang2/app-client/src/components/common/test.vue'' < ''/tmp/nvim2NLUn5/4/test.vue''']

<<<OUTPUT STARTS>>>
[{"filePath":"/home/pwang2/app-client/src/components/common/test.vue","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}]
<<<OUTPUT ENDS>>>

**Comment by hsanson** [comment]
There no trace of cocvetur or anything similar to it in ALE source. I do not think this has anything to do with ALE.

**Comment by pwang2** [comment]
The command line message '[cocvetur] eslint:xxxxxxx' is a format I configured for ALE.  And I have disabled lsp for ALE.  If I call ALEDisable, the message will disappear.  I agree in the ALEInfo there in no trace, but it is absolutely related to ALE here. 

Update: I moved away from coc.vim and switched to nvim native LSP, so far so good.:)

**Comment by lancekrogers** [comment]
I'm having a similar issue with ALE as well with python.  After making a change to a file and saving, ALE displays import errors and formatting errors that dont make sense, along with many error indicators that aren't accompanied by an error explanation and are not listed in ALE detail.  All the errors disappear when I run :ALEDisable.  So far I've just fixed it by putting vim in the background and then bringing it back to the foreground, and that solves the issue briefly, but certainly not a permanent fix.

**Comment by pwang2** [comment]
@lancekrogers you might want to include your details here to make a deeper investigation.   I moved away from coc.nvim and it seems less confusing and obsessing to me now.
